### DISCUSSION TITLE
Training data is lagging behind

### DISCUSSION THREAD

**Discussion by regolas** [community/community#77451] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

- VS Code Version: 1.84.2
- OS Version: Win 11 22H2
- Logs:
Since ,NET 8 is GA, why does the training set go back to .NET 6. In the case of Maui, it has changed quite a bis since .NET6.

Steps to Reproduce:

1. I asked the question "in maui will the tab control support multi row tabs"
2. The response from Copilot "GitHub Copilot: As of the current version of .NET MAUI (**as of .NET 6**), the TabView control does not natively support multi-row tabs. The tabs in a TabView are designed to be displayed in a single row.

If you need multi-row tabs, you would need to create a custom control or use a third-party control that supports this feature. 

Alternatively, you could consider redesigning your UI to work with the single-row constraint of the TabView control. For example, you could use a combination of tabs and other navigational elements such as a ListView or CollectionView to organize your content.

For the next steps, you might want to:
- Explore third-party libraries that offer multi-row tab controls.
- Consider redesigning your UI to work with the single-row constraint of the TabView control.
- Look into creating a custom control that supports multi-row tabs."

**Comment by outgaze** [comment]
Cause they don't want to give us turbo 😭😭😭😭
